pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -88,10 +88,17 @@ func (op *overcommitPlugin) OnSessionOpen(ssn *framework.Session) {
 	}
 	op.idleResource = total.Clone().Multi(op.overCommitFactor).Sub(used)
 
-	// calculate inqueue job resources
 	for _, job := range ssn.Jobs {
+		// calculate inqueue job resources
 		if job.PodGroup.Status.Phase == scheduling.PodGroupInqueue && job.PodGroup.Spec.MinResources != nil {
 			op.inqueueResource.Add(api.NewResource(*job.PodGroup.Spec.MinResources))
+			continue
+		}
+		// calculate inqueue resource for running jobs
+		if job.PodGroup.Status.Phase == scheduling.PodGroupRunning && job.PodGroup.Spec.MinResources != nil {
+			allocated := util.GetAllocatedResource(job)
+			inqueued := util.GetInqueueResource(job, allocated)
+			op.inqueueResource.Add(inqueued)
```

Comment:
Add the consideration for this case, Spark app is driver pod is completed and podgroup is running.
In this case, the inqueued should be `0` . The judgement might be as below.
job.PodGroup.Status.Phase == scheduling.PodGroupRunning && jobInfo.PodGroup.Spec.MinMember > `allocated numbers in job`

Reply:
Yes, I think it is more suitable to add a phase `Completed` for podgroup.

Reply:
IC, that's right. Let me modify the condition.

Reply:
Yes. new `state` needs to be added for `podgroup`. Right now the hot fix here is ok for me.  Please add detail comments here. We will refactor it later.

Reply:
Yes, podgroup `completed` state is necessary. Currently, in addition to vcjob which deletes the pg when it completes, the pg created by third-party job will not be deleted after the task is completed(though AFAIK spark operator [managed it itself](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/v1beta2-1.3.3-3.1.1/pkg/batchscheduler/volcano/volcano_scheduler.go#L164)), but will remain in inqueue or running, which causes unnecessary inqueue computations.

Reply:
But there is a detail l want to confirm, I think `job.PodGroup.Status.Running >= job.PodGroup.Spec.MinMember` may should be replaced by `job.TaskStatusIndex[api.Running] >= job.PodGroup.Spec.MinMember`?

Reply:
you are right